recipes/nm.rb

```diff
@@ -19,12 +19,53 @@
 end
 
 
+cloud="false"
+if node['cloud'] == "true" || node['install']['localhost'].casecmp("true") == 0 
```

Comment:
@jimdowling where is this `node['cloud']` being set?

Reply:
It's being set in conda-chef/attributes/default.rb
If basically checks if install/aws|gce|azure is true, and if yes, sets it to true.